src/pages/LandingPage.vue

```diff
@@ -1,13 +1,91 @@
 <template>
-  <ProductList />
+  <div class="container">
+    <div class="navigation">
+      <v-button @click="showModal = !showModal">
+        Add Item
+      </v-button>
+    </div>
+    <Modal
+      v-if="showModal"
+      @close="showModal = !showModal"
+    >
+      <div slot="header">
+        <p>{{ action }}</p>
+      </div>
+      <template slot="body">
+        <add-edit-form 
+          :item="item"
+          :action="action"
+          @close="showModal = !showModal"
+          @save-item="saveNewItem"
+        />
+      </template>
```

Comment:
Just a note, that you can define slot on any element and if you don't need to wrap multiple elements in a slot, consider just defining a slot on a given element.
```suggestion
      <add-edit-form 
        slot="body"
        :item="item"
        :action="action"
        @close="showModal = !showModal"
        @save-item="saveNewItem"
      />
```